### DISCUSSION TITLE
Hierarchy Time Series Prediction Model

### DISCUSSION THREAD

**Discussion by hanlianzf** [awslabs/gluonts#1678] [General]
Hi, friends, have seen your reference, you have referenced paper "End-to-End Learning of Coherent Probabilistic Forecasts for Hierarchical Time Series", but i have not find the model in repo, and btw the paper said it has implemented on Gluonts, so I want to make sure whether it has been pushed or not, thanks.

**Comment by mbohlkeschneider** [comment]
Hi @hanlianzf, 

thank you for your interest! The PR of the model is here: https://github.com/awslabs/gluon-ts/pull/1665. @rshyamsundar is still iterating on it and we will merge it as soon as it is ready.

**Reply by hanlianzf** [reply]
oh, it has provide a lot of help to us, thank you very much.
